frontend/src/components/Contributions/ContributionForm.vue

```diff
@@ -50,15 +50,27 @@
           <span v-for="error in errors" class="errors" :key="error">{{ error }}</span>
         </b-col>
       </validation-provider>
-      <label class="mt-3">{{ $t('form.amount') }} {{ $t('math.asterisk') }}</label>
-      <b-input-group size="lg" prepend="GDD">
-        <b-form-input
-          id="contribution-amount"
-          v-model="form.amount"
-          type="text"
-          :formatter="numberFormat"
-        ></b-form-input>
-      </b-input-group>
+      <validation-provider
+        :name="$t('form.amount')"
+        :rules="{
+          required: true,
```

Comment:
Use the same validation mechanism as in `components/GddSend/TransactionForm.vue`
```
                :rules="{
                  required: true,
                  gddSendAmount: [0.01, balance],
                }"
```

Reply:
added